ooniprobe/Utility/TestUtility.mm

```diff
@@ -144,7 +144,23 @@ + (void)downloadUrlsCallback:(NSData *)data
     successcb(urls);
 }
 
-+ (void)removeFile:(NSString*)fileName {
++ (void)deleteUploadedJsonsWithMeasurementRemover:(void (^)(Measurement *))remover {
+    for (Measurement *measurement in [Measurement measurementsWithJson]) {
+        [measurement getExplorerUrl:^(NSString *measurement_url){
+            remover(measurement);
+        } onError:^(NSError *error) {
+        }];
+    }
+}
+
+//TODO where and when to call it?
```

Comment:
I guess this comment is redundant because we will do that in the next PR

Reply:
Actually not, the next PR is to open a link.
We need to decide here where to call this function
1) On app opening (always/once a week/etc...)
2) On app resuming (every time you go from background to foreground)
3) When opening raw data (a local json) always checking if that test has been uploaded

I think 1 and 3.

Reply:
👍 thanks, I was missing that bit